src/components/pages/rules/flyouts/ruleNew/ruleNew.js

```diff
@@ -184,23 +274,23 @@ export class RuleNew extends LinkedComponent {
                   <Radio
                     placeholder="critical"
                     checked={this.state.severityLevel.critical}>
-                    <SeverityRenderer value="Critical" context={{ t }} iconOnly={false} />
+                    <SeverityRenderer value={'Critical'} context={{ t }} iconOnly={false} />
```

Comment:
Are these magical strings? Perhaps add constants somewhere?

Reply:
no need for constants since we are not reusing them anywhere else

Reply:
fixed